app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Media Ranker 2.0</title>
+
+  <%= stylesheet_link_tag    "application", "https://fonts.googleapis.com/css?family=Gudea:400,700|Hammersmith+One", media: "all", 'data-turbolinks-track' => true %>
+  <%= favicon_link_tag(source='owl.jpg', options={}) %>
+  <%= javascript_include_tag "application", 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+</head>
+
+<body>
+
+
+  <header class="app-header__header">
+    <h1>
+      <a href="/">Media Ranker</a>
```

Comment:
Instead of using anchor tags, use `link_to ` and named routes.  It makes things easier to adjust.